src/taskpane/pages/citationStyle.tsx

```diff
@@ -0,0 +1,129 @@
+import * as React from "react";
+import { FocusZone, FocusZoneDirection } from "@fluentui/react/lib/FocusZone";
+import { List } from "@fluentui/react/lib/List";
+import {
+  ITheme,
+  mergeStyleSets,
+  getTheme,
+  getFocusStyle,
+} from "@fluentui/react/lib/Styling";
+import { Stack } from "@fluentui/react";
+
+const theme: ITheme = getTheme();
+const { palette, semanticColors, fonts } = theme;
+const classNames = mergeStyleSets({
+  container: {
+    overflow: "auto",
+    padding: "0.25rem 0.25rem 0px",
+    webkitBoxFlex: "1 1 auto",
+  },
+  itemCell: [
+    getFocusStyle(theme, { inset: -1 }),
+    {
+      minHeight: 34,
+      boxSizing: "border-box",
+      padding: "0 25px",
+      display: "flex",
+      selectors: {
+        "&:hover": { background: palette.themeLighterAlt },
+      },
+    },
+  ],
+  itemName: [
+    fonts.medium,
+    {
+      padding: "8px 0",
+      borderBottom: `1px solid ${semanticColors.bodyDivider}`,
+      minWidth: "100%",
+    },
+  ],
+  StyleHeading: [
+    fonts.medium,
+    {
+      color: palette.neutralSecondary,
+      fontWeight: "bold",
+      padding: 20,
+      paddingBottom: 5,
+      paddingTop: 15,
+    },
+  ],
+  selectedStyle: [
+    fonts.medium,
+    {
+      color: palette.themeDarkAlt,
+      padding: 20,
+      paddingBottom: 5,
+      paddingTop: 5,
+      fontWeight: "bold",
+    },
+  ],
+});
+
+function CitationStyle(): JSX.Element {
+  const items = [
+    {
+      text: "American Political Science Association",
+      value: "american-political-science-association",
+    },
+    { text: "IEEE", value: "ieee" },
+    {
+      text: "American Sociological Association 6th edition",
+      value: "american-sociological-association",
+    },
+    {
+      text: "American Psychological Association 7th edition",
+      value: "advances-in-complex-systems",
+    },
+    {
+      text: "Chicago Manual of Style 16th edition (author-date)",
+      value: "chicago-author-date-16th-edition",
+    },
+  ];
+  const preferenceStyle: string | null = Office.context.document.settings.get(
```

Comment:
I would propose to extract all the settings related code to a new `CitationStyleSettings` class. For now this would only include methods to get/set the currently selected style (and a wrapper around `saveAsync`). This allows you to handle. default values etc in a central place.

This may develop into something more complex in the future, e.g. similar to https://github.com/ichenzhifan/office/blob/d4afd0f0304f8b675e775475e1eaefcde2cb15f8/ChartAgave/ChartAgaveWeb/App/Core/ChartSettings.ts